### PR TITLE
feat: [sc-143546] Python Client Lib Update for Verify APIs and removing Company and EmailVerification support

### PR DESCRIPTION
**Pull request by MananMukim**
Story details: https://app.shortcut.com/fullcontact/story/143546

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@
 ![pytest](https://github.com/fullcontact/fullcontact-python-client/workflows/pytest/badge.svg)
 
 The official python client library for FullContact V3 API. This client provides an interface to interact with Enrich,
-Resolve, Tags, Audience and Verification APIs. FullContact API Documentation is available
+Resolve, Tags, Audience APIs. FullContact API Documentation is available
```
Comment (prakhar-j) [inline README.md]: We can add that it supports Verify and Permission APIs

File: README.md [hunk 2]
```diff
@@ -1651,3 +1457,243 @@ All these takes same parameters as their synchronous counterparts but return a `
...
+
+The client library provides methods to interact with V3 Verfiy API (`verify.match`, `verify.signals` and `verify.activity` 
+endpoints) through `FullContactClient.verify` object. The V3 Verify API can be accessed using
+the methods [match()](#fullcontactclientverifymatch), [signals()](#fullcontactclientverifysignals) and 
+[activity()](#fullcontactclientverifyactivity), respectively. These APIs can be accessed using the async version these
+functions, [match_async()](#fullcontactclientverifymatch_async), [signals_async()](#fullcontactclientverifysignals_async) and
+[activity_async()](#fullcontactclientverifyactivity_async). Additional headers can be set on a per-request basis by
+setting the parameter `headers` while calling these methods.     
+Being a request level parameter, this can be used to override any header that has been set on the client level.
+> Verify API Documentation: https://docs.fullcontact.com/docs/multi-field-request-verify
```
Comment (prakhar-j) [inline README.md]: Link can be https://docs.fullcontact.com/docs/verify-overview

File: fullcontact/__about__.py [hunk 3]
```diff
@@ -8,7 +8,7 @@
 ]
 
 __name__ = "python-fullcontact"
-__version__ = "3.1.0"
+__version__ = "3.2.0"
```
Comment (prakhar-j) [inline fullcontact/__about__.py]: Version should be 4.0.0, as it has breaking changes

**Review by prakhar-j — CHANGES_REQUESTED** [review]
Some minor comments, Rest looks good. Thanks!

File: README.md [hunk 4]
```diff
@@ -390,7 +390,7 @@ class: _fullcontact.response.person_response.PersonEnrichResponse_
 * `get_finance()`: _dict_ - Finance details from Person Enrich Response
 * `get_census()`: _dict_ - Census details from Person Enrich Response
 * `get_identifiers()`: _dict_ - Identifiers from Person Enrich Response
-* `get_epsilon()`: _dict_ - All Epsilon data
+* `get_extended()`: _dict_ - All Epsilon data
```
Comment (prakhar-j) [inline README.md]: Please replace with: "All Extended data"

**Review by prakhar-j — CHANGES_REQUESTED** [review]

**Review by jidler — APPROVED** [review]

**Pull request merged**